pkg/sql/pgwire/testdata/pgtest/timezone

```diff
@@ -5,8 +5,8 @@ Query {"String": "SET TIME ZONE \"UTC\""}
 until
 ReadyForQuery
 ----
-{"Type":"ParameterStatus","Name":"TimeZone","Value":"UTC"}
 {"Type":"CommandComplete","CommandTag":"SET"}
+{"Type":"ParameterStatus","Name":"TimeZone","Value":"UTC"}
```

Comment:
i have just discovered PG doesn't send this out if the value of the timezone didn't change, but that's not a now problem...